rules/rules-reviewed/eap8/eap7/hibernate6.windup.xml

```diff
@@ -0,0 +1,530 @@
+        </rule>
+        <rule id="hibernate6-00160">
+            <when>
+                <xmlfile matches="//*/property[count(column) > 1]" in="hbm.xml"/>
+            </when>
+            <perform>
+                <hint title="Support for basic property mappings with multiple columns was removed" category-id="mandatory" effort="1">
+                    <message>In Hibernate 6.0 the support for basic property mappings with multiple columns was removed. The only use case for that was when a `CompositeUserType` was in use, which was reworked to now work on top of components.</message>
+                    <link href="https://github.com/hibernate/hibernate-orm/blob/6.0/migration-guide.adoc#hbmxml-multiple-column-now-disallowed" title="Hibernate ORM 6 migration guide - hbm.xml multiple &lt;column/&gt; now disallowed"/>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="hibernate6-00170">
+            <when>
+                <javaclass references="org.hibernate.criterion.{*}">
+                    <location>IMPORT</location>
+                </javaclass>
+            </when>
+            <perform>
+                <hint title="Legacy Hibernate Criteria API has been removed" category-id="mandatory" effort="1">
+                    <message>The legacy Hibernate Criteria API which was deprecated back in Hibernate 5.x and removed in 6.0. Usually, all queries using the legacy API can be modeled with the JPA Criteria API. In some cases it is necessary to use the Hibernate JPA Criteria extensions.</message>
+                    <link href="https://github.com/hibernate/hibernate-orm/blob/6.0/migration-guide.adoc#legacy-hibernate-criteria-api" title="Hibernate ORM 6 migration guide - Legacy hibernate criteria API"/>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="hibernate6-00180">
+            <when>
+                <javaclass references="org.hibernate.query.Query.iterate({*})">
+                    <location>METHOD_CALL</location>
+                </javaclass>
+            </when>
+            <perform>
+                <hint title="The Query#iterate() method has been removed" category-id="mandatory" effort="1">
+                    <message>The Query#iterate() method has been removed. See link for alternatives.</message>
+                    <link href="https://github.com/hibernate/hibernate-orm/blob/6.0/migration-guide.adoc#iterate" title="Hibernate ORM 6 migration guide - Iterate"/>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="hibernate6-00190">
+            <when>
+                <or>
+                    <javaclass references="org.hibernate.annotations.NamedNativeQuery">
+                        <location>ANNOTATION</location>
+                        <annotation-literal name="callable" pattern="true"/>
+                    </javaclass>
+                    <xmlfile matches="sql-query[@callable = 'true']" in="hbm.xml"/>
```

Comment:
The `matches` is missing, at least, the `//*/` suffix.
I've checked the report generated in tests and this condition doesn't trigger even if `hbm.xml` has the `<sql-query name="query" callable="true">` tag.